README.rst

```diff
@@ -0,0 +1,183 @@
+* The rest of the modules and plugins will move into multiple Collections
+* Collections can:
+
+  * Be release independently of Ansible, ie no more waiting 6 months for your new feature
+  * Have their own repo (GitHub, GitLab, etc) with dedicated backlog, ie no more shared massive issue & PR backlog
+  * Should still have CI testing
+  
+* Ansible 2.10 will pull in ``ansible-base`` and the various community collections.
+
+
+Terminology
+===========
+
+
+Classic Ansible (CA)
+  What we think of as Ansible prior to 2020. The open source project and the deliverable. What you currently get (as of Ansible 2.9) when you type “pip install ansible”. Currently developed almost entirely within the ``ansible/ansible`` repository on GitHub. 
+  
+Collection
+  A packaging format for bundling and distributing Ansible content: plugins, roles, modules. Can be released independently to other collections or Ansible so features can be made available sooner to users. Installed via ``ansible-galaxy collection install <namespace.collection>``.
+
+Ansible Base
+  The codebase that will be contained in github.com/ansible/ansible for the Ansible 2.10 release. Contains only the minimal amount of modules and plugins, allows other collections to be installed. Similar to Ansible 2.9 though without any content that has since moved into a collection. Only contains a small set of modules and plugins. A work in progress can be found in `ansible-base <https://github.com/ansible-collection-migration/ansible-base/>`_ repository.
+
+
+Ansible Community Content Collections
+  The collections which host the modules and plugins, previously in Ansible 2.9, which are managed by the community. These collections will be combined and released when you do ``$package_manager install ansible``. These may run at different cadences, and may now be installed individually by users via Ansible Galaxy.
+
+Ansible Galaxy
+  An online hub for finding and sharing Ansible community content.  Also, the command-line utility that lets users  install individual Ansible Collections. `galaxy.ansible.com <https://galaxy.ansible.com/>`_.
+
+Fully Qualified Collection Name (FQCN)
+  The full definition of a module, plugin, or role hosted within a collection, in the form ``namespace.collection.content_name``. Allows a Playbook to refer to a specific module or plugin from a specific source in an unambiguous manner, for example, ``community.grafana.grafana_dashboard``. The FQCN is required when you want to specify the exact source of a module and multiple modules with the same name are available. Can always be identified in a playbook; ideally not necessary in most playbooks, but in cases in which users have multiple collections installed with similar content, the FQCN will always be the explicit and authoritative indicator of which collection to use for content. Example: cisco.ios.ios_config would be the FQCN, and the playbook would generally call "ios_config". When is this required.
+
```

Comment:
I assume the last sentence 'When is this required' is meant to be a placeholder/TODO for writing details of when FQCN is required?